README.md

```diff
@@ -326,36 +309,32 @@ The official Meteor resources page can be found [here](https://www.meteor.com/to
 ## Courses
 
 * [EventedMind](https://www.eventedmind.com/)
+* [MeteorTuts](http://meteor-tuts.com/)
+* [BulletProof Meteor](https://bulletproofmeteor.com/) - Online course for Meteor performance in production, by Arunoda Susiripala
 * [Udemy — Meteor: Build a real-time web app using only JavaScript](https://www.udemy.com/meteor-course/)
 * [Udemy - A Beginner's Guide to the Meteor JavaScript Framework](https://www.udemy.com/meteorjs/)
 * [tuts+ - Single Page Web Apps with Meteor](http://code.tutsplus.com/courses/single-page-web-apps-with-meteor)
-* [DevFreeCasts.org](http://devfreecasts.org/meteor/)
-* [MeteorTuts](http://meteor-tuts.com/)
 
 ## Tutorials
 
 * [Building a CMS-powered blog in Meteor](https://buttercms.com/blog/meteor-cms-blog-tutorial)
 * [scotch.io - Building a Slack Clone in Meteor](https://scotch.io/tutorials/building-a-slack-clone-in-meteor-js-getting-started)
-* [Rocket-Chat Slack Clone](https://rocket.chat/)
 * [Meteor Learning](https://github.com/ericdouglas/Meteor-Learning) - List of resources to learn
 * [Phusion Passenger: Meteor tutorial](https://github.com/phusion/passenger/wiki/Phusion-Passenger:-Meteor-tutorial)
+* [When a Meteor finally hits production](https://medium.com/@davidyahalomi/when-a-meteor-finally-hits-production-6c37b81f795b) - Blog post about deploying Meteor apps
+
 
 ## Blogs
 
 * [Official Meteor blog](http://blog.meteor.com)
-* [The Meteor podcast](http://podcast.crater.io)
```

Comment:
why?

Reply:
A big reason behind this PR was to modernize the resources and as such, there is no need to have deprecated resources. Latest episode was published at January 6th, 2017!

Reply:
true. I wish there were new ones.  but do you think listening to the old ones is not beneficial anymore?
It might give great context on where we are and the original goals and such.
I'll leave it for your choice